ui/js/values.js

```diff
@@ -101,6 +101,11 @@ treeherder.value("phAlertSummaryStatusMap", {
     BACKEDOUT: { id: 8, text: "backedout" }
 });
 
+treeherder.value("phAlertSummaryIssueTrackersMap", {
```

Comment:
It would be good to remove this hard coding at some point soon (doesn't necessarily need to be in this PR) - since at the moment there are no tests to ensure the chosen `id` matches what's defined on the backend.

Reply:
I'm filing a bug for that.

Reply:
> I'm filing a separate bug for that.

Great - thank you :-)